### PR TITLE
Dala 385 version updates sprint 23

### PR DESCRIPTION
**Pull request by MarcTM01**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one E2E Test exists testing the new feature
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] The new version is deployed to the dev server using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [ ] There is at least one picture for each story, which was created before coding has started
- [x] If any work on the UI is to be merged, those changes were also documented in the Figma 
- [x] The local Dev stack still works: execute `startDevelopmentStack.sh`, npm serve, npm teste2e and execute Cypress Tests locally

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -12,21 +12,21 @@ dependencyResolutionManagement {
...
+            library("swagger-gradle-plugin", "io.swagger.core.v3:swagger-gradle-plugin:2.2.2")
+            library("swagger-annotations", "io.swagger.core.v3:swagger-annotations:2.2.2")
 
-            library("okhttp", "com.squareup.okhttp3:okhttp:4.9.3")
+            library("okhttp", "com.squareup.okhttp3:okhttp:4.10.0")
             library("rs-api", "jakarta.ws.rs:jakarta.ws.rs-api:3.1.0")
 
-            library("skyminder-client", "org.dataland:skyminder-client:0.1.6")
+            library("skyminder-client", "org.dataland:skyminder-client:0.1.7")
             library("dataland-edc-client", "org.dataland:dataland-edc-client:0.2.3")
```
Comment (ppuetsch) [inline settings.gradle.kts:30]: 0.2.5

Comment (MarcTM01) [reply settings.gradle.kts:30]: Ja. Aber 0.2.5 ist dann auch schon mit der neuen EDC-Version. Das ist ja noch on hold und wird dann in DALA-297 gemacht oder nicht?

File: README.md [hunk 2]
```diff
@@ -73,10 +73,11 @@ To do so:
 * update `settings.gradle.kts` (for libraries), `build.gradle.kts` (for plugins) and `gradle.properties` (for jacoco)
 * Note: fasterXML is managed by spring and ktlint by jlleitschuh, thus NO manual version update should be conducted
 * update the gradle wrapper: execute `gradle wrapper --gradle-version X.Y.Z`
-* Update Fronted packages: run the `updatepackages` script, e.g. by  `npm run updatepackages` to update versions in package.json  
+* * Update node version in `dataland-frontend/build.gradle.kts`
```
Comment (ppuetsch) [inline README.md:76]: nicht `* *` - `*` reicht.

File: build.gradle.kts [hunk 3]
```diff
@@ -47,23 +47,23 @@ tasks.dependencyUpdates.configure {
...
-    id("com.github.node-gradle.node") version "3.3.0" apply false
-    id("org.springframework.boot") version "2.7.1" apply false
+    id("com.github.node-gradle.node") version "3.4.0" apply false
+    id("org.springframework.boot") version "2.7.2" apply false
     id("org.jlleitschuh.gradle.ktlint") version "10.3.0"
     kotlin("jvm") version "1.7.10"
     kotlin("plugin.spring") version "1.7.10" apply false
     id("org.sonarqube") version "3.4.0.2513"
     jacoco
     id("org.springdoc.openapi-gradle-plugin") version "1.3.4" apply false
```
Comment (ppuetsch) [inline build.gradle.kts:64]: 1.4.0

File: gradlew [hunk 4]
```diff
@@ -205,6 +205,12 @@ set -- \
         org.gradle.wrapper.GradleWrapperMain \
         "$@"
 
+# Stop when "xargs" is not available.
```
Comment (ppuetsch) [inline gradlew:208]: gradlew 7.5.1

File: dataland-frontend/package.json [hunk 5]
```diff
@@ -20,40 +20,40 @@
     "checkdependencies": "depcheck --ignores='@vue/cli-plugin-*,@vue/vue3-jest,sass-loader,ts-jest'"
   },
   "dependencies": {
-    "@formkit/themes": "^1.0.0-beta.9",
-    "@formkit/vue": "^1.0.0-beta.9",
+    "@formkit/themes": "^1.0.0-beta.10",
+    "@formkit/vue": "^1.0.0-beta.10",
```
Comment (ppuetsch) [inline dataland-frontend/package.json:24]: bitte die package-lock noch einmal update

File: dataland-keycloak/dataland_theme/login/package.json [hunk 6]
```diff
@@ -3,13 +3,18 @@
...
+    "updatepackagelock": "npm update --save --dev && npm audit fix"
   },
   "dependencies": {
     "@ibm/plex": "^6.0.2",
-    "material-icons": "^1.11.5",
+    "material-icons": "^1.11.8",
     "primeflex": "^3.2.1",
     "primeicons": "^5.0.0",
-    "sass": "^1.53.0"
+    "sass": "^1.54.2"
```
Comment (ppuetsch) [inline dataland-keycloak/dataland_theme/login/package.json:15]: Kannst du noch einen Abschnitt in die readme schreiben, um diese package/package-lock zu updaten?

Comment (MarcTM01) [reply dataland-keycloak/dataland_theme/login/package.json:15]: Ich habe bereits in der Readme dazu vermerkt: "Do this in the frontend as well as in the keycloak theme".

File: build.gradle.kts (hunk 3 above)
Comment (MarcTM01) [reply build.gradle.kts:64]: Jup. Mache ich. Die 1.4.0 gabs letzten Freitag noch nicht.

File: gradlew (hunk 4 above)
Comment (MarcTM01) [reply gradlew:208]: Jup. Mache ich. Die 7.5.1 gabs letzten Freitag noch nicht.

File: README.md (hunk 2 above)
Comment (MarcTM01) [reply README.md]: Done

File: build.gradle.kts (hunk 3 above)
Comment (MarcTM01) [reply build.gradle.kts]: Done

File: gradlew (hunk 4 above)
Comment (MarcTM01) [reply gradlew:208]: Done

File: dataland-frontend/package.json (hunk 5 above)
Comment (MarcTM01) [reply dataland-frontend/package.json:24]: Done

File: settings.gradle.kts (hunk 1 above)
Comment (ppuetsch) [reply settings.gradle.kts:30]: ok

File: build.gradle.kts (hunk 3 above)
Comment (ppuetsch) [reply build.gradle.kts]: ok

File: gradlew (hunk 4 above)
Comment (ppuetsch) [reply gradlew:208]: ok

File: README.md (hunk 2 above)
Comment (ppuetsch) [reply README.md]: ok

File: dataland-keycloak/dataland_theme/login/package.json (hunk 6 above)
Comment (ppuetsch) [reply dataland-keycloak/dataland_theme/login/package.json]: ok

File: dataland-frontend/package.json (hunk 5 above)
Comment (ppuetsch) [reply dataland-frontend/package.json:24]: ok

**Pull request merged**